.github/workflows/ci.yml

```diff
@@ -0,0 +1,392 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      # Feature branches beginning with "feature."
+      - feature.*
+    tags:
+      - '**'
+  pull_request:
+
+jobs:
+  sass_spec:
+    runs-on: ubuntu-latest
+
+    name: "sass-spec | Dart ${{ matrix.dart_channel }} | ${{ matrix.async_label }}"
+
+    strategy:
+      fail-fast: false
+      matrix:
+        dart_channel: [stable, dev]
+        async_label: [synchronous]
+        async_args: ['']
+        include:
+          - dart_channel: stable
+            async_label: asynchronous
+            async_args: '--cmd-args --async'
+
+    steps:
+      - name: Checkout
```

Comment:
Style nit: for configuration-file brevity, I prefer to leave off step names when they're redundant with the name of the action repo itself.